frontend/src/lib/auth/index.ts

```diff
@@ -1,13 +1,19 @@
-import crypto from "crypto";
+import CryptoJS, { HmacSHA1 } from "crypto-js";
```

Comment:
dlaczego używasz crypto-js zamiast crypto

Reply:
tak jak wyzej, next ma problem do crypto (bardzo to wkurzajace) ale nic nie poradze